plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7580,11 +7580,13 @@ public void testSelectFromPrestoViewReferencingHiveTableWithTimestamps()
 
```

Comment:
@losipiuk For testcase in 7574-7576, I think there's another issue too (independent to already linked). View analysis uses catalog from the stored definition, so while reading the table `ts_hive_table_xxx`, the catalog config-properties will be applied for `hive` and not `hive_timestamp_nanos`. So the type derived for the table will still be `timestamp(3)`. So I'm not sure if "should be" is the desired output. thoughts?

```
// TODO(https://github.com/prestosql/presto/issues/6295) Presto view schema is fixed on creation
// should be: assertThat(query(defaultSession, "SELECT ts  FROM hive_timestamp_nanos.tpch." + prestoViewNameDefault)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123456789'")
assertThat(query(defaultSession, "SELECT ts  FROM hive_timestamp_nanos.tpch." + prestoViewNameDefault)).matches("VALUES TIMESTAMP '1990-01-02 12:13:14.123'");
```

Reply:
Thanks for pointing that out. I do not recall exactly - but based on what I wrote I assume I was not aware of that. So maybe the comment is not needed here. Both semantics kinda make sense but it feels like using session properties for source catalog is more natural. So assertion is fine here.